_posts/2020-05-13-cockpit-219.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 219
+author: kkoukiou
+date: 2020-05-13
+tags: cockpit linux
+slug: cockpit-219
+category: release
+summary: Cockpit with XYZ
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 219.
+
+### Logs: Improved journal logs filtering
```

Comment:
I think "Improved filtering" suffices, the prefix already makes it clear that it's about the Logs.